managed/src/main/java/com/yugabyte/yw/models/InstanceType.java

```diff
@@ -29,6 +31,9 @@
 
   public static List<String> AWS_INSTANCE_PREFIXES_SUPPORTED = ImmutableList.of(
     "m3.", "c5.", "c5d.", "c4.", "c3.", "i3.");
+  static final String YB_VOLUME_INFO_VOLUME_COUNT = "yb.volumeInfo.volume_count";
+  static final String YB_VOLUME_INFO_VOLUME_SIZE_GB = "yb.volumeInfo.volume_size_gb";
+  private static ConfigFactory config;
```

Comment:
why `static`?

Reply:
changed config var to non-static, others kept as it is, since those are being used in static block.

Reply:
yep. my comment was only about config.